apps/vercel/src/locations/ConfigScreen.tsx

```diff
@@ -0,0 +1,118 @@
+import { useCallback, useState, useEffect, ChangeEvent } from 'react';
+import { ConfigAppSDK } from '@contentful/app-sdk';
+import {
+  Box,
+  Flex,
+  FormControl,
+  Heading,
+  HelpText,
+  Paragraph,
+  Stack,
+  TextInput,
+  TextLink,
+} from '@contentful/f36-components';
+import { useSDK } from '@contentful/react-apps-toolkit';
+import { ExternalLinkIcon } from '@contentful/f36-icons';
+import { styles } from '../components/config-screen/ConfigScreen.styles';
+import VercelIcon from '../components/common/VercelIcon';
+
+export interface AppInstallationParameters {
+  accessToken?: string;
+}
+
+const ConfigScreen = () => {
+  const sdk = useSDK<ConfigAppSDK>();
+  const [parameters, setParameters] = useState<AppInstallationParameters>({});
```

Comment:
Not necessary for this current PR, but we may want to switch this over to a reducer rather than state. 

That way we can easily utilize field validation and have a centralized place for managing state.

Reply:
yeah intention was not to over complicate this since we are moving towards na OAUTH flow, right now the only requirements are to be able to store the access token in parameters and using a controlled input to manage the value